src/Common/nunit/ColorConsole.cs

```diff
@@ -76,7 +76,6 @@ public static ConsoleColor GetColor(ColorStyle style)
                             return ConsoleColor.DarkGray;
                         case ColorStyle.Help:
                             return ConsoleColor.DarkGray;
-                        case ColorStyle.Default:
                         default:
```

Comment:
There's nothing wrong with your choice to take out that line. There's nothing wrong with the author's choice to leave in that line. It's just you doing stylistic changes that are not actually covered by the coding standard.

Since this PR is titled "Code cleanup in line with Dev wiki coding standards" I'm pointing out that this is not such a change. I'll post more at length on the list after I go through these.